public/app/features/dashboard/api/ResponseTransformers.ts

```diff
@@ -66,6 +70,20 @@ export function ensureV2Response(
       }
     : dto.meta;
 
+  const metaAnnotations: GrafanaAnnotations & GrafanaClientAnnotations = {
```

Comment:
Alternative approach, not sure which one i like more :)

```suggestion
  const metaAnnotations: DashboardWithAccessInfo<DashboardV2Spec>['metadata']['annotations'] = {
```

Reply:
I like this alternative approach, it's clearer and saves on importing :)